spec/app/dependency_file_updaters/ruby_spec.rb

```diff
@@ -82,7 +82,7 @@
       let(:gemfile_body) { fixture("gemfiles", "version_not_specified") }
 
       it "locks the updated gem to the latest version" do
-        expect(file.content).to include "business (1.5.0)"
+        expect(file.content).to include "business (1.7.0)"
```

Comment:
Sadly this test depends on the current version of `business` 😢 

A battle for another day, I think